### PR TITLE
Fix Doorbell.co to Doorbell.io

### PR DESCRIPTION
**Pull request by manavo**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -97,9 +97,9 @@
                 <iron-icon icon="done"></iron-icon>
               </paper-icon-item>
               <paper-icon-item>
-                <div class="avatar doorbell" item-icon></div>
```
Comment (zonito) [inline templates/index.html:100]: this is required, please revert

File: templates/index.html [hunk 2]
```diff
@@ -97,9 +97,9 @@
...
+                <div class="avatar " item-icon></div>
                 <paper-item-body two-line>
-                  <div>Doorbell.io</div>
```
Comment (zonito) [inline templates/index.html:102]: ? This is as well required

File: templates/index.html (hunk 1 above)
Comment (manavo) [inline templates/index.html:100]: Really sorry, have no idea how I messed this up! Will fix it now!

File: templates/index.html (hunk 2 above)
Comment (manavo) [inline templates/index.html:102]: Really sorry, have no idea how I messed this up! Will fix it now!

**Comment by manavo** [comment]
Fixed this, really sorry for breaking it @zonito!

**Comment by zonito** [comment]
Thanks @manavo for quick fix. :)

**Pull request merged**